spec/features/guide_spec.rb

```diff
@@ -236,6 +236,30 @@
     end
   end
 
+  describe "slug generation" do
```

Comment:
perhaps add `js: true` here, because you've forgotten it for the test on lines 254-260 :)